ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-feature.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  centos8:
+    name: centos
+    namespace: openshift
+    tag: stream8
+  centos9:
+    name: centos
+    namespace: openshift
+    tag: stream9
+  shellcheck:
+    name: shellcheck
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: "1.21"
+images:
+- context_dir: images/prow-image
+  dockerfile_path: Dockerfile
+  from: centos9
+  to: baremetal-qe-base-feature
```

Comment:
hi @bmanzari, can we make this the same as before and change the tag at line 27 to 'staging'?